liquibase-standard/src/main/java/liquibase/diff/output/changelog/DiffToChangeLog.java

```diff
@@ -836,6 +846,30 @@ public void setChangeSetPath(String changeSetPath) {
         this.changeSetPath = changeSetPath;
     }
 
+    public void setChangeSetRunOnChangeTypes(final String[] runOnChangeTypes) {
+        changeSetRunOnChangeTypes = runOnChangeTypes;
+    }
+
+    protected String[] getChangeSetRunOnChangeTypes() {
+        return changeSetRunOnChangeTypes;
+    }
+
+    private boolean isContainedInRunOnChangeTypes(final Change change) {
```

Comment:
It seems this method is causing some tests to fail. I ran **GenerateChangeLogMySQLIntegrationTest** and some tests and cleanup spec method are failing with below NPE:

```
liquibase.exception.CommandExecutionException: liquibase.exception.DatabaseException: java.lang.NullPointerException

	at liquibase.command.CommandScope.execute(CommandScope.java:237)
	at liquibase.command.util.CommandUtil.runDropAll(CommandUtil.groovy:112)
	at liquibase.command.core.GenerateChangeLogMySQLIntegrationTest.Ensure generated changelog map JSON columns correctly(GenerateChangeLogMySQLIntegrationTest.groovy:113)
Caused by: liquibase.exception.DatabaseException: java.lang.NullPointerException
	at liquibase.command.core.DropAllCommandStep.run(DropAllCommandStep.java:75)
	at liquibase.command.CommandScope.execute(CommandScope.java:214)
	... 2 more
Caused by: java.lang.NullPointerException
	at java.base/java.util.Objects.requireNonNull(Objects.java:221)
	at java.base/java.util.Arrays$ArrayList.<init>(Arrays.java:4323)
	at java.base/java.util.Arrays.asList(Arrays.java:4310)
	at liquibase.diff.output.changelog.DiffToChangeLog.isContainedInRunOnChangeTypes(DiffToChangeLog.java:858)
	at liquibase.diff.output.changelog.DiffToChangeLog.lambda$addToChangeSets$15(DiffToChangeLog.java:776)
	at java.base/java.util.stream.MatchOps$1MatchSink.accept(MatchOps.java:90)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:127)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:230)
	at java.base/java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:196)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.allMatch(ReferencePipeline.java:533)
	at liquibase.diff.output.changelog.DiffToChangeLog.addToChangeSets(DiffToChangeLog.java:776)
	at liquibase.diff.output.changelog.DiffToChangeLog.generateChangeSets(DiffToChangeLog.java:327)
	at liquibase.database.AbstractJdbcDatabase.dropDatabaseObjects(AbstractJdbcDatabase.java:815)
	at liquibase.command.core.DropAllCommandStep.run(DropAllCommandStep.java:65)
	... 3 more
```
Could you please take a look at this?

Reply:
Thank you for reporting this! Indeed I forgot to perfroam a `null` check. I have just fixed that in c87d9eac5e3d0b2d16a3162a6468846436c2af52.